Chummer/frmCharacterOptions.cs

```diff
@@ -765,11 +814,50 @@ private void PopulateCustomDataDirectoryTreeView()
                 }
             }
 
-            if(objOldSelected != null)
+
+            if (objOldSelected != null)
                 treCustomDataDirectories.SelectedNode = treCustomDataDirectories.FindNodeByTag(objOldSelected);
+            treCustomDataDirectories.ShowNodeToolTips = true;
             treCustomDataDirectories.EndUpdate();
         }
 
+        private void CheckForDependenciesAndExclusivities()
+        {
+            foreach (TreeNode directoryNode in treCustomDataDirectories.Nodes)
+            {
+                if (!(directoryNode.Tag is CustomDataDirectoryInfo customDataDirectory)) continue;
+                if (directoryNode.Checked)
+                {
+                    // check dependencies and exclusivities only if they could exist at all instead of calling and running into empty an foreach.
+                    string missingDirectories = string.Empty;
+                    if (customDataDirectory.DependenciesList.Count>0)
+                        missingDirectories = customDataDirectory.CheckDependency(_objCharacterOptions);
+
+                    string prohibitedDirectories = string.Empty;
+                    if (customDataDirectory.ExclusivitiesList.Count>0)
+                        prohibitedDirectories = customDataDirectory.CheckExclusivity(_objCharacterOptions);
+
+                    if (string.IsNullOrEmpty(missingDirectories) && string.IsNullOrEmpty(prohibitedDirectories))
+                    {
+                        directoryNode.ToolTipText = string.Empty;
+                        directoryNode.ForeColor = Color.Empty;
```

Comment:
Should this be `Color.Empty`? What color is that in practice, and how does it interact with `ColorManager` and dark mode?

Reply:
I will chek how this interacts with the dark mode, but in theory this should just remove the additional color from the node.